pkg/client/client.go

```diff
@@ -89,6 +89,28 @@ func (c *Client) HandleChannel(channel string, handler func(ctx context.Context,
 	})
 }
 
+// SendRouteRequest receives the http request informations and send it to the sidecar server
+func (c *Client) SendRouteRequest(ctx context.Context, route, path, method string, body interface{}) (interface{}, error) {
+	l := logger.With(zap.String("operation", "sendRequest"), zap.String("route", route))
+
+	var resp interface{}
+	// sends a message to the corresponding route on the sidecar
+	l.Debug("sending message to load balancer")
+	err := c.client.Send(
+		ctx,
+		fmt.Sprintf("/route/%s/%s", route, path),
```

Comment:
Why do you need route and path here? In fact, whats the difference between route and path? In my vision, the user will call this function using all the path, like SendRouteRequest(ctx, "/mypath/myendpoint" , ...)